usaspending_api/disaster/management/sql/disaster_covid19_file_d1_awards.sql

```diff
@@ -1,3 +1,13 @@
+WITH recent_submission AS (
```

Comment:
This CTE doesn't actually return the recent submission, but all "closed" submissions

Reply:
10-4